### PR TITLE
Homework 4

### PR DESCRIPTION
**Pull request by Kuzeikind**
Test refactored with the steps pattern.

### CODE REVIEW

**Review by akulebyakin — CHANGES_REQUESTED** [review]
Review comments please

File: src/test/java/com/epam/tc/hw4/steps/HomePageSteps.java [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+import com.epam.tc.hw3.components.leftside.LeftSideMenu;
+import com.epam.tc.hw3.components.navbar.NavigationBarMenu;
+import com.epam.tc.hw3.util.ComponentUtil;
+import com.epam.tc.hw3.util.PageData;
+import io.qameta.allure.Step;
+import java.util.List;
+import org.openqa.selenium.WebDriver;
+import org.testng.asserts.SoftAssert;
+
+public class HomePageSteps {
```
[the remark this answers is not in the document]
Comment (Kuzeikind) [reply src/test/java/com/epam/tc/hw4/steps/HomePageSteps.java:13]: Fixed the issue.

File: src/test/java/com/epam/tc/hw4/steps/DifferentElementsPageSteps.java [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+    public DifferentElementsPageSteps(WebDriver driver) {
+        super(driver);
+    }
+
+    @Step
+    public void assertDifferentElementsPageIsOpened() {
+        homePage.getLeftSideMenu().getServicesMenu().selectItemByName("Different elements");
+        differentElementsPage = new DifferentElementsPage(driver);
+        assertions.assertEquals(differentElementsPage.getTitle(), PageData.elementsPageTitle);
+    }
```
[the remark this answers is not in the document]
Comment (Kuzeikind) [reply src/test/java/com/epam/tc/hw4/steps/DifferentElementsPageSteps.java]: Added proper description to the tests.

**Review by akulebyakin — APPROVED** [review]
Approved. Good job!

**Pull request merged**